backend/python/tests/coverage.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
```

Comment:
to stay consistent, let's use bash instead
```suggestion
#!/bin/bash
```

Reply:
you may have to toggle the `CRLF/LF` if you get a bad interpreter message:
![image](https://user-images.githubusercontent.com/32009013/178151246-9d09a2e2-23be-42a7-bf9f-66d3b2e6fe4a.png)